test/User.spec.ts

```diff
@@ -1,17 +1,34 @@
 import { MilvusClient } from '../milvus';
-
-import { IP } from '../const';
+import { GENERATE_NAME, IP } from '../const';
 import { ERROR_REASONS } from '../milvus/const/ErrorReason';
 import { ErrorCode } from '../milvus/types/Response';
 import { timeoutTest } from './common/timeout';
+import { DataType, Roles } from '../milvus/types/Common';
+import { genCollectionParams } from '../utils/test';
 
 let milvusClient = new MilvusClient(IP);
 let authClient: MilvusClient | null = null;
 const USERNAME = 'nameczz';
 const PASSWORD = '123456';
 const NEW_PASSWORD = '1234567';
+const ROLENAME = GENERATE_NAME('role');
+const COLLECTION_NAME = GENERATE_NAME();
 
 describe('User Auth Api', () => {
+  beforeAll(async () => {
+    authClient = new MilvusClient(IP, false, USERNAME, NEW_PASSWORD);
+    await authClient.collectionManager.createCollection(
+      genCollectionParams(COLLECTION_NAME, '4', DataType.FloatVector, false)
+    );
+  });
+
+  afterAll(async () => {
+    authClient = new MilvusClient(IP, false, USERNAME, NEW_PASSWORD);
```

Comment:
i think we just need to init authClient in beforeAll function is enough.
Then we don't need to connect and close in all spec functions?